modules/snooker/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Fixed
+- `CopySelected` was incorrectly extracting a selected cells table for tables with colgroups.
```

Comment:
This doesn't read well, or I can't seem to make sense of it. Can we please have a second go at the wording here?

Reply:
Maybe something like this?
```suggestion
- The `CopySelected` module was incorrectly extracting the selected cells in a table that contained colgroups.
```